README-rust-release.md

```diff
@@ -18,9 +18,21 @@ The [Rust crates at @stellar][crates] are released using the following process:
 
   - [1. Bump Version](#1-bump-version)
   - [2. Review PR](#2-review-pr)
-  - [3. Merge PR](#3-merge-pr)
+  - [3. Merge PR](#3-merge-pr) (for `main` branch releases)
   - [4. Create Release on GitHub](#4-create-release-on-github)
-  - [5. Monitor the Publish](#5-monitor-the-publish)
+  - [5. Close PR](#5-merge-pr) (for backport patch releases)
+  - [6. Monitor the Publish](#6-monitor-the-publish)
+
+Note that this release process leans into trunk-based releasing, where most
+releases occur directly from the `main` branch. If patches need to be made to
+past releases:
+1. Create a release by performing step 1 below.
```

Comment:
Step 1 below is bump-version, it does not create a release. I'm not sure how I'd follow these instructions.

Reply:
I reworded this point and the section. Can you take another look?